superduperdb/vector_search/lancedb_client.py

```diff
@@ -82,14 +83,28 @@ def get(self, identifier: str, limit: int = 1) -> t.List[t.Any]:
         vector = vector_df[VECTOR_FIELD_NAME]
         return vector
 
+    def size(self) -> int:
+        """
+        Get the number of rows in ``LanceTable``.
+        """
+        # substract 1 for the seed vector
+        return len(self.table) - 1
+
     def add(self, data: t.Sequence[VectorCollectionItem], upsert: bool = False) -> None:
         """
         Add vectors to the ``LanceTable``.
 
         :param data: t.Sequence of ``VectorCollectionItem`` objects.
         :param upsert: Whether to perform an upsert operation. Defaults to ``False``.
         """
-        dict_data = [d.to_dict() for d in data]
+        dict_data = []
+        for d in data:
+            dict_d = d.to_dict()
+            vector = dict_d['vector']
+            if isinstance(vector, torch.Tensor):
```

Comment:
@blythed Lancedb do not support torch tensors, hence I had to convert explicitly
let me know if the above seems alrite to you